main.ts

```diff
@@ -77,6 +148,13 @@ async function run(): Promise<void> {
       '/usr/bin',
       `/${mingw.toLocaleLowerCase()}/bin`
     ]
+
+    if (architecture === 'aarch64') {
+      // Some binaries aren't available yet in the /clangarm64/bin folder, but Windows 11 ARM64
+      // has support for x64 emulation, so let's add /mingw64/bin as a fallback.
+      binPaths.splice(binPaths.length - 2, 0, '/mingw64/bin')
```

Comment:
This results in the following array:

```JS
[ '/usr/bin/core_perl', '/mingw64/bin', '/usr/bin', '/clangarm64/bin' ]
```

... which will eventually end up in the path as follows:

```
/clangarm64/bin:/usr/bin:/mingw64/bin:/usr/bin/core_perl:${PATH}
```

Is it intended to put `/usr/bin` before `/mingw64/bin` in this case? I always thought it's preferred to put `/mingw64` first and fallback to `/usr/bin` only if really needed, but I lack any clear background info/reasoning on that. No strong preference on my side for this one - just curious!

Reply:
Oh no, this is an off-by-one... 😱

Reply:
@dennisameling could I bother you to test again?

Reply:
@dscho thanks! I also tested with `makepkg-git` instead of `full` this time, and with `cleanup: true`. Everything is working as expected 🎉 https://github.com/dennisameling/git/actions/runs/3159629157/jobs/5143099298